djvm-example/settings.gradle

```diff
@@ -5,3 +5,11 @@ pluginManagement {
 }
 
 rootProject.name = "djvm-example"
+
+if (file('../djvm').isDirectory()) {
```

Comment:
This seems unrelated to this change?

Reply:
Umm, well, possibly :wink:. Technically yes, but this and SLF4J initialisation are fixing the common _underlying_ issue that Joel's example functions aren't working correctly in the independent sub-project.

Reply:
To be fair, the changes to `archiveBaseName` in the `build.gradle` file are related to this Gradle Composite Build change too... :speak_no_evil: